src/plugins/data/server/index_patterns/routes.ts

```diff
@@ -70,7 +70,21 @@ export function registerRoutes(http: HttpServiceSetup) {
           },
         });
       } catch (error) {
-        return response.notFound();
+        if (
+          !!error.isBoom &&
```

Comment:
does `error` need a `?` here? (since it has one below)

Reply:
Yes, added.